cfgov/v1/models/sublanding_page.py

```diff
@@ -41,6 +41,7 @@ class SublandingPage(CFGOVPage):
         ('formfield_with_button', molecules.FormFieldWithButton()),
         ('reg_comment', organisms.RegComment()),
         ('feedback', v1_blocks.Feedback()),
+        ('snippet_list', organisms.SnippetList()),    
```

Comment:
Thanks for catching that, @Scotchester -- fixed in https://github.com/cfpb/cfgov-refresh/pull/3131/commits/f06d4f00dd92e5b07ebe3e5a386036b1ac25f5b6.